tests/test_batoid_wcs.py

```diff
@@ -597,8 +598,46 @@ def test_config():
     np.testing.assert_raises(NotImplementedError,
                              galsim.config.BuildWCS, config['image'], 'wcs', config)
 
+def test_focal_to_pixel() -> None:
+    det = imsim.get_camera()[0]
+    x = np.linspace(start=-190.615, stop=-188.615, num=4)
+    y = np.linspace(start=-317.255, stop=-315.255, num=4)
+    x_t, y_t = batoid_wcs.focal_to_pixel(
+        np.tile(x, len(y)), np.repeat(y, len(x)), det
+    )
+
+    x_expected = y_expected = np.linspace(start=-100.0, stop=100.0, num=4)
+    np.testing.assert_almost_equal(x_t, np.tile(x_expected, len(y)))
+    np.testing.assert_almost_equal(y_t, np.repeat(y_expected, len(x)))
+
+
+def test_pixel_to_focal() -> None:
+    det = imsim.get_camera()[0]
+    x = y = np.linspace(start=-100.0, stop=100.0, num=4)
+    x_t, y_t = batoid_wcs.pixel_to_focal(
+        np.tile(x, len(y)), np.repeat(y, len(x)), det
+    )
+
+    x_expected = np.linspace(start=-190.615, stop=-188.615, num=4)
+    y_expected = np.linspace(start=-317.255, stop=-315.255, num=4)
+    np.testing.assert_almost_equal(x_t, np.tile(x_expected, len(y)))
+    np.testing.assert_almost_equal(y_t, np.repeat(y_expected, len(x)))
+
```

Comment:
Need some kind of test of the new photon operator.

Also, you seem to have added an `__init__.py` file in the tests directory, which doesn't belong here.

Reply:
The `__init__.py` file turns the `tests` dir into a python module. This simplifies test discovery and python module discovery.
For example, a `python -m unittest tests/test_batoid_wcs.py` would only work with an `__init__.py` in the `tests` dir, as otherwise, you would get `ModuleNotFoundError: No module named 'tests.test_batoid_wcs'`

Reply:
OK. Just thought that adding a `__init__.py` would not hurt and enable `unittest` support. Note, that this practice is also described here: https://docs.pytest.org/en/6.2.x/goodpractices.html
If you want me to remove it again, I ll of course do that.

Reply:
Also note: https://docs.python.org/3/library/unittest.html#test-discovery
(for the case we want to stick with `unittest`)
And I also saw, that my boilerplate code is not sufficient for test discovery. I would need to do something like:
https://docs.python.org/3/library/unittest.html#load-tests-protocol

Reply:
If it were up to me, I'd rip out all the unittest stuff, and just have test functions, calling them with either pytest or nosetests, not the unittest module.  That's by far my preferred structure for test suites.
But as you noted, that's not what we seem to be mostly doing in imsim so far.  So I'll defer to @jchiang87's opinion on this, since I think he made the initial structure here.

Reply:
Do you have a suggestion for a test for the photon op?
Test some very simple batoid telescope?

Reply:
`__init__.py` and other unittest boilerplate removed in the tests for `batoid_wcs` and the new photon op